### PR TITLE
Ks/#subscription doc cleanup

### PR DESCRIPTION
**Pull request by KSchopmeyer**
modify the doc/pybemcli/commands.rst to correct issues found in reviews of the documentation.

See commit for more details

### CODE REVIEW

File: docs/pywbemcli/commands.rst [hunk 1]
```diff
@@ -2288,7 +2288,7 @@ The ``connection show`` command shows information about a connection definition:
 
 .. code-block:: text
 
-    pywbemcli -s http://blah connection show
+    pywbemcli> -s http://blah connection show
```
Comment (andy-maier) [inline docs/pywbemcli/commands.rst:2291]: If you want to show the interactive prompt, you should also show the command to get there, like in the `connection select` command, above.

File: docs/pywbemcli/commands.rst [hunk 2]
```diff
@@ -2858,8 +2880,9 @@ The command line options for this command are:
 * ``--source-namespaces`` The name of WBEM namespaces on the server where
   indications originate.  Multiple namespaces may be defined as one or
   more namespaces per options instantiation where multiple namespaces are
-  separated by comma (","). For example, ``--source-namespaces
-  root/cimv2,root/cimv3"" or ``--source-namespaces root/cimv2 --source-namespaces root/cimv3``.
+  separated by comma (","). For example either, ``--source-namespaces
+  root/cimv2,root/cimv3"" or ``--source-namespaces root/cimv2 --source-namespaces root/cimv3``
```
Comment (andy-maier) [inline docs/pywbemcli/commands.rst:2884]: Change double quotes after cimv3 to backquotes.

File: docs/pywbemcli/commands.rst [hunk 3]
```diff
@@ -2969,7 +3011,8 @@ The options are:
 
 .. code-block:: text
 
-    $ subscription list
+    pywbemcli> subscription list
+
```
Comment (andy-maier) [inline docs/pywbemcli/commands.rst:3015]: Same issue a few lines down.

File: docs/pywbemcli/commands.rst [hunk 4]
```diff
@@ -2999,6 +3044,12 @@ the current WBEM server.
 class so that any indication destination instances defined with the superclass
 ``CIM_ListenerDestination`` are not visible.
 
+The format of this command is:
+
+.. code-block:: text
+
+    pywbemcli [GENERAL-OPTIONS] subscription list-destinations [COMMAND-OPTIONS]
```
Comment (andy-maier) [inline docs/pywbemcli/commands.rst:3051]: For commands in the shell, show a `$` prompt.

**Review by andy-maier — CHANGES_REQUESTED** [review]
Some comments.

File: docs/pywbemcli/commands.rst (hunk 1 above)
Comment (KSchopmeyer) [reply docs/pywbemcli/commands.rst:2291]: Fixed a bunch of them.

File: docs/pywbemcli/commands.rst (hunk 2 above)
Comment (KSchopmeyer) [reply docs/pywbemcli/commands.rst:2884]: DONE

File: docs/pywbemcli/commands.rst (hunk 3 above)
Comment (KSchopmeyer) [reply docs/pywbemcli/commands.rst:3015]: DONE

File: docs/pywbemcli/commands.rst (hunk 4 above)
Comment (KSchopmeyer) [reply docs/pywbemcli/commands.rst:3051]: Fixed those.

**Review by andy-maier — APPROVED** [review]

**Pull request merged**